plugins/content/createmenulink/createmenulink.php

```diff
@@ -163,7 +164,9 @@ public function onContentAfterSave($context, $article)
 		$data    = $session->get("formData");
 		$session->clear("formData");
 
-		list($component, $view) = explode('.', $context);
+		$jinput    = JFactory::getApplication()->input;
+		$component = $jinput->get('option');
+		$view      = $jinput->get('view');
```

Comment:
@gunjanpatel Under the function oncontentpreparedata() , If I use `$jinput    = JFactory::getApplication()->input;` this code this returns both option and view as com_content and article...
But under the function oncontentaftersave() , if i use `$jinput    = JFactory::getApplication()->input;` it rturns only option not view...
What is wrong here?